src/shippingservice/quote.go

```diff
@@ -50,6 +50,6 @@ func quoteByCountFloat(count int) float64 {
 		return 0
 	}
 	count64 := float64(count)
-	var p float64 = 1 + (count64 * 0.2)
+	var p = 1 + (count64 * 0.2)
```

Comment:
I'm not sure what this line changes. isn't it already float64?

Reply:
`var p` doesn't require `float64` type definition as it's inferred from `count64`. Moreover now I realized it doesn't require `var` keyword as well.

`p := 1 + (count64 * 0.2)` just works.